css/styles.css

```diff
@@ -0,0 +1,5 @@
+#best-offers .content .model .nav li {
```

Comment:
Irving, your commit cause conflict with styles.css file, please update your local and compile again the styles, then push again, thank you.